plugins/synaptics-mst/fu-synaptics-mst-device.c

```diff
@@ -1627,9 +1628,18 @@ fu_synaptics_mst_device_setup(FuDevice *device, GError **error)
 	locker = fu_device_locker_new_full(self,
 					   (FuDeviceLockerFunc)fu_synaptics_mst_device_enable_rc,
 					   (FuDeviceLockerFunc)fu_synaptics_mst_device_disable_rc,
-					   error);
-	if (locker == NULL)
+					   &error_local);
+	if (locker == NULL) {
+		if (g_strcmp0(fu_device_get_name(device), "DPMST") == 0) {
+			g_set_error_literal(error,
+					    G_IO_ERROR,
+					    G_IO_ERROR_NOT_SUPPORTED,
+					    "downstream endpoint not supported");
```

Comment:
do we want to return with an error, or just skip the locker completely for DPMST devices?

Reply:
I don't think we want to be continuing setup unless we could communicate with the device.

Reply:
Ohh I figured it was just the RC that was failing -- and the other stuff worked without it. I've got a horrible feeling we might need to do the RC manually on the root node to be able to upgrade the DPMST device. In the meantime, this seems okay to me.

Reply:
I don't think there are **really** two Synaptics MST hubs connected in this bug.  But once we have a setup with two connected cascaded we can see what's **really** needed.